gobblin-yarn/src/main/java/gobblin/yarn/GobblinWorkUnitRunner.java

```diff
@@ -174,6 +162,7 @@ public GobblinWorkUnitRunner(String applicationName, String helixInstanceName, C
       LOGGER.info("Adding YarnContainerSecurityManager since login is keytab based");
       services.add(new YarnContainerSecurityManager(config, fs, this.eventBus));
     }
+    services.add(new JMXReportingService());
```

Comment:
We could be there are services like the Job History Store and Admin UI that aren't really needed. Also the Metrics Service would be different as `GobblinWorkUnitRunner` uses `ContainerMetrics`